gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/ExportPanel.java

```diff
@@ -391,9 +428,14 @@ private File buildFile(final boolean isCustomExport) {
 					if (isCustomExport && this.includeSectionMembership) {
 						line.add((userSections.size() > 0) ? userSections.get(0) : getString("sections.label.none"));
 					}
-					if (!isCustomExport || this.includeGradeItemScores || this.includeGradeItemComments) {
-						assignments.forEach(assignment -> {
-							final GbGradeInfo gradeInfo = studentGradeInfo.getGrades().get(assignment.getId());
+					if (!isCustomExport || this.includeGradeItemScores || this.includeGradeItemComments || this.includeCategoryAverages) {
+						final Map<Long, Double> categoryAverages = studentGradeInfo.getCategoryAverages();
```

Comment:
Maybe this should be moved to after line 458 as it's not used

Reply:
yes but then it would be in the assignment loop!